src/variables.jl

```diff
@@ -236,7 +262,10 @@ constraint if one exists, otherwise create a
 new one.
 """
 function JuMP.fix(vref::LogicalVariableRef, value::Bool)
-    new_var = LogicalVariable(value, JuMP.start_value(vref))
+    if has_logical_compliment(vref)
+        error("Cannot fix value of a variable with a logical compliment.")
```

Comment:
yet?

Reply:
This is to avoid conflicts. I clarified the error.